### DISCUSSION TITLE
Conditionally disable GestureDetector?

### DISCUSSION THREAD

**Discussion by Nantris** [software-mansion/react-native-gesture-handler#3161] [Q&A] [answered]
I need to disable `GestureDetector` because under certain circumstances it's interfering with handling of gestures inside of a Webview it wraps, but it seems like there's no method that can prevent `Gesture.Pinch` from interfering with the Webview except unmounting it, which would mean unmounting the webview too which we can't do.

**Comment by Glazzes** [comment, marked as answer]
All gestures have a method called enabled, you can enable a gesture dynamically like this:
```js
const [isEnabled, setIsEnabled] = useState<boolean>(true);

const pinch = Gesture.Pinch().enabled(isEnabled);
```

**Reply by Nantris** [reply]
Thanks for your reply, and right you are. Unfortunately our issue ended up being more deep-rooted than that. #3196
